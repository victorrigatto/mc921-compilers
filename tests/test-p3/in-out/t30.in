void a() {
   return;
}

int main() {
    print(a());
    return 0;
}
