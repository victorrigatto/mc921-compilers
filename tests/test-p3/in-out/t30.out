SemanticError: Expression is not of basic type @ 6:11
